app/code/Magento/UrlRewriteGraphQl/Model/Resolver/EntityUrl.php

```diff
@@ -0,0 +1,161 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\UrlRewriteGraphQl\Model\Resolver;
+
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Query\Resolver\Value;
+use Magento\Framework\GraphQl\Query\Resolver\ValueFactory;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Store\Model\StoreManagerInterface;
+use Magento\UrlRewrite\Model\UrlFinderInterface;
+use Magento\UrlRewriteGraphQl\Model\Resolver\UrlRewrite\CustomUrlLocatorInterface;
+
+/**
+ * UrlRewrite field resolver, used for GraphQL request processing.
```

Comment:
As EntityUrl and UrlRewrite are pretty similar it is better to describe their purpose in class comments (now both classes have same description so to understand what is needed source code should be explored)